public/scripts/app.js

```diff
@@ -1,7 +1,82 @@
-console.log("Sanity Check: JS is working!");
+$(document).ready(function() {
+
+// ajax get request to get data from database and render to client side
+$.ajax({
+	method:"GET",
+	url: '/api/bjj',
+	success: renderMultipleReviews
+});
 
-$(document).ready(function(){
 
-// your code
+  function formData(e){
+    e.preventDefault();
+    var formData = $(this).serialize();
+    console.log('formData', formData);
+   $.post('/api/bjj', formData).done(renderMultipleReviews);
+    window.location = window.location; 
+  }
 
+// form submission event handler
+$('.form-horizontal').on('submit', formData);
+$('#gymDisplay').on("click", ".deleteBtn", handleDeleteBjj);
 });
+
+
+
+// loop through data and render to html
+function renderMultipleReviews(reviews) {
+console.log(reviews);
+  reviews.forEach(function(review) {
+  renderBjj(review);
+  });
+
+}
+
+// data render to html function
+function renderBjj(bjj) {
+
+
+  var bjjHtml = (`
+  
+<div class="container forms" data-bjj='${bjj._id}'>
+<div class="row">
+<div class="col-sm-12">
+<div class="card" style="width: 75%;">
```

Comment:
manually setting width is so sad! could add another class so you can still use class-based styling for these cards.